draft-birkholz-scitt-software-supply-chain-use-cases.md

```diff
@@ -96,18 +118,30 @@ The following steps are performed by the Software Producer:
 1. Create the “final SBOM” listing all the components contained in the final distribution package of a software product, which the customer will install into their environment.
 The SBOM must follow NTIA minimum elements for SBOM’s and other NTIA and NIST recommendations for SBOM, to meet Executive Order 14028 and OMB M-22-18 requirements.
 (Co)SWID, SPDX or CycloneDX SBOM formats are acceptable for this artifact. 
-2. Digitally sign the SBOM artifact.
-3. Place the SBOM and digital signature artifacts within an access-controlled location, i.e. a customer portal, and provide the end consumer with a link to these artifacts for downloading to the customers environment.
+2. The encoding of SBOM is specific to the protocol in question.
+3. Makes a claim to establish its own identity as a producer of the overall SBOM.
+4. The issuer in certain situations can make further claims (example claims that relate to steps that might assist independent building of the software components, to arrive at the final deliverable).
```

Comment:
This leaves a lot to the imagination. Why would all these claims clumped together and not be... stand-alone claims?